product/pom.xml

```diff
@@ -10,19 +10,15 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-	<maven.compiler.source>11</maven.compiler.source>
-	<maven.compiler.target>11</maven.compiler.target>
+	<maven.compiler.source>8</maven.compiler.source>
```

Comment:
narayana master no longer supports Java 8 so I think this needs to either remain at 11 or we need to add a comment here and in the associated issue number. The comment should also indicate the plan for resolving whatever issue is blocking Java 11 support.

Reply:
Sorry, that is my mistake. It does work with JDK11

Reply:
I remove this change then.